pages/common/git-obliterate.md

```diff
@@ -0,0 +1,13 @@
+# git obliterate
+
+> Remove a file from all history in a Git repository.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-obliterate>.
+
+- Remove all existence of a file including history:
```

Comment:
How about:
```suggestion
- Delete files without a trace including their history:
```

Reply:
Yes I like that but is sounds wrong.
Here:

```suggestion
- Delete files and traces of that file:
```

The trace would be their history

Reply:
Perhaps this will read better?
```suggestion
- Remove a file from the current repository's history:
```

Reply:
Yet onother suggestion:
```suggestion
- Delete files and their history:
```